src/win/process.c

```diff
@@ -1250,7 +1253,87 @@ static int uv__kill(HANDLE process_handle, int signum) {
     return UV_EINVAL;
   }
 
+  /*
+   * Create a dump file for the targeted process, if the registry key
+   * `HKLM:Software\Microsoft\Windows\Windows Error Reporting\LocalDumps`
+   * exists.  The location of the dumps can be influenced by the `DumpFolder`
+   * sub-key, which has a default value of `%LOCALAPPDATA%\CrashDumps`, see [0]
+   * for more detail.  Note that if the dump folder does not exist, we attempt
+   * to create it, to match behavior with WER itself.
+   * [0]: https://learn.microsoft.com/en-us/windows/win32/wer/collecting-user-mode-dumps
+   */
+  if (signum == SIGQUIT) {
+    /* Get target process name. */
+    WCHAR basename[MAX_PATH];
+    GetModuleBaseNameW(process_handle, NULL, &basename[0], sizeof(basename));
+
+    /* Get PID of target process. */
+    DWORD pid = GetProcessId(process_handle);
```

Comment:
style nit: don't mix code and decl on the same line (but also keep all decls together)
```suggestion
    DWORD pid;
    pid = GetProcessId(process_handle);
```